src/main/java/org/kairosdb/core/datastore/KairosDatastore.java

```diff
@@ -444,6 +450,9 @@ public int getSampleSize()
 		@Override
 		public List<DataPointGroup> execute() throws DatastoreException
 		{
+
+			Span span = tracer.buildSpan("query_database_datapoints_count").start();
```

Comment:
startActive(...) - otherwise if we have to instrument lower levels this span will not propagate.

Should probably also try/finally this with the span.finish() far below at the end of the method.